shopify/customized-bundle-template/web/components/BundleCard.tsx

```diff
@@ -40,48 +39,84 @@ export default ({
   price,
   bundleComponents,
   bundleComponentCount,
+}: {
+  id: string;
+  title: string;
+  description: string | null;
+  status: Status;
+  price: number | null;
+  bundleComponents: {
```

Comment:
It would be nice if we could make use of the BundleComponent type defined in the BundleForm

Reply:
This is also not the same type. Not sure why